# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/qiv/00-config.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-05-22 03:21:47.000000000 -0400
+++ ./Makefile	2005-02-11 19:03:45.689653447 -0300
@@ -4,7 +4,7 @@
 #######################################################################
 
 # Directory where qiv will be installed under.
-PREFIX = /usr/local
+PREFIX = /usr
 
 # Font to use for statusbar in fullscreen mode
 STATUSBAR_FONT = "fixed"
@@ -14,10 +14,10 @@
 CURSOR = 84
 
 # Should image be centered on screen by default? 1=yes, 0=no.
-CENTER = 1
+CENTER = 0
 
 # Should images be filtered by extension? 1=yes, 0=no.
-FILTER = 1
+FILTER = 0
 
 # This sets the file extentions to filter on (other file types will be
 # skipped.) It should reflect whatever is compiled into imlib.
@@ -28,7 +28,7 @@
 
 # This program will be run on the manual page after it is installed.
 # If you don't want to compress the manpage, change it to 'true'.
-COMPRESS_PROG = gzip -9f
+COMPRESS_PROG = true
 
 # Comment this line out if your system doesn't have libXinerama
 # installed (for centering on dual-screen)
@@ -126,11 +126,6 @@
 	@echo "Installing QIV..."
 	install -s -m 0755 $(PROGRAM) $(PREFIX)/bin
 	install -m 0644 $(PROGRAM).1 $(PREFIX)/man/man1
-	$(COMPRESS_PROG) $(PREFIX)/man/man1/$(PROGRAM).1
-	@if ./qiv -f ./intro.jpg ; \
-	then echo "-- Test Passed --" ; \
-	else echo "-- Test Failed --" ; \
-	fi
 	@echo -ne "\nDont forget to look into the \"qiv-command\" file and install it!\n-> cp qiv-command.example /usr/local/bin/qiv-command\n\n" 
 
 install-xscreensaver: install
